IABot/API.php

```diff
@@ -773,7 +773,12 @@ public static function getAllArticles( $limit, $resume ) {
 	* @copyright Copyright (c) 2016, Maximilian Doerr
 	* @return mixed Revid if successful, else false
 	*/
-	public static function edit( $page, $text, $summary, $minor = false, $timestamp = false, $bot = true, $section = false, $title = "", &$error = null ) {
+	public static function edit( $page, $text, $summary, $minor = false, $timestamp = false, $bot = true, $section = false, $title = "", &
+		$error = null ) {
+		if( TESTMODE ) {
+			echo $text;
```

Comment:
We essentially just want to do a loop with comparisons of bot output and what it should output. Creating a file is doable but is unnecessary overhead for this purpose. E.g. `TESTMODE` is strictly for our acceptance tests